spec/javascripts/app/models/person_spec.js

```diff
@@ -1,69 +1,69 @@
+    this.blockedContact = factory.person({relationship: "blocked", block: {id: 1}});
   });
 
   context("#isSharing", function() {
     it("indicates if the person is sharing", function() {
-      expect(this.mutual_contact.isSharing()).toBeTruthy();
-      expect(this.sharing_contact.isSharing()).toBeTruthy();
+      expect(this.mutualContact.isSharing()).toBeTruthy();
+      expect(this.sharingContact.isSharing()).toBeTruthy();
 
-      expect(this.receiving_contact.isSharing()).toBeFalsy();
-      expect(this.blocked_contact.isSharing()).toBeFalsy();
+      expect(this.receivingContact.isSharing()).toBeFalsy();
+      expect(this.blockedContact.isSharing()).toBeFalsy();
     });
   });
 
   context("#isReceiving", function() {
     it("indicates if the person is receiving", function() {
-      expect(this.mutual_contact.isReceiving()).toBeTruthy();
-      expect(this.receiving_contact.isReceiving()).toBeTruthy();
+      expect(this.mutualContact.isReceiving()).toBeTruthy();
+      expect(this.receivingContact.isReceiving()).toBeTruthy();
 
-      expect(this.sharing_contact.isReceiving()).toBeFalsy();
-      expect(this.blocked_contact.isReceiving()).toBeFalsy();
+      expect(this.sharingContact.isReceiving()).toBeFalsy();
+      expect(this.blockedContact.isReceiving()).toBeFalsy();
     });
   });
 
   context("#isMutual", function() {
     it("indicates if we share mutually with the person", function() {
-      expect(this.mutual_contact.isMutual()).toBeTruthy();
+      expect(this.mutualContact.isMutual()).toBeTruthy();
 
-      expect(this.receiving_contact.isMutual()).toBeFalsy();
-      expect(this.sharing_contact.isMutual()).toBeFalsy();
-      expect(this.blocked_contact.isMutual()).toBeFalsy();
+      expect(this.receivingContact.isMutual()).toBeFalsy();
+      expect(this.sharingContact.isMutual()).toBeFalsy();
+      expect(this.blockedContact.isMutual()).toBeFalsy();
     });
   });
 
   context("#isBlocked", function() {
     it("indicates whether we blocked the person", function() {
-      expect(this.blocked_contact.isBlocked()).toBeTruthy();
+      expect(this.blockedContact.isBlocked()).toBeTruthy();
 
-      expect(this.mutual_contact.isBlocked()).toBeFalsy();
-      expect(this.receiving_contact.isBlocked()).toBeFalsy();
-      expect(this.sharing_contact.isBlocked()).toBeFalsy();
+      expect(this.mutualContact.isBlocked()).toBeFalsy();
+      expect(this.receivingContact.isBlocked()).toBeFalsy();
+      expect(this.sharingContact.isBlocked()).toBeFalsy();
     });
   });
 
   context("#block", function() {
     it("POSTs a block to the server", function() {
-      this.sharing_contact.block();
+      this.sharingContact.block();
       var request = jasmine.Ajax.requests.mostRecent();
 
       expect(request.method).toEqual("POST");
-      expect($.parseJSON(request.params).block.person_id).toEqual(this.sharing_contact.id);
+      expect($.parseJSON(request.params).block.person_id).toEqual(this.sharingContact.id);
```

Comment:
This is not in camel case because it is data which is sent to the server. Again: won't fix.